notebooks/text_classification/solutions/word2vec.ipynb

```diff
@@ -0,0 +1,1453 @@
+{
```

Comment:
<p>This may require more instructions. </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/asl-ml-immersion/pull/154/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='114'/>

Reply:
<p>Instead of (or additionally to) analyzing the embeddings with tensorbaord, it may be useful to write code to retrieve the embeddings and use them into a Keras model.</p>

Reply:
<p>Updated the instructions. For the Keras model, I want to have a separate lab for that, just because this lab is already quite long and with subclassing, it will be a lot of learning.</p>